src/plays/smart-todo-list/styles.css

```diff
@@ -0,0 +1,334 @@
+:root {
+  --clr-primary: #0ce7f3;
+  --clr-secondary: #010326;
+  --clr-light: #fff;
+  --clr-active: #33ae10;
+  --clr-complete: #5454e8;
+  --clr-update: #00d866;
+  --clr-delete: #ff7849;
+}
+#wa_todo-list {
+  position: relative;
+  max-width: 768px;
+  margin: 0 auto;
+  font-size: 14px;
+}
+
+.wrapper {
```

Comment:
I would suggest a unique name as prefix to this style to avoid accidental overrides.

Reply:
Fixed